extern/ycm/CMakeLists.txt

```diff
@@ -86,7 +86,8 @@ set(YCM_FOUND ${YCM_FOUND} PARENT_SCOPE)
 foreach(_d ycm-0.6.0   # since YARP 3.0.0
            ycm-0.7.0
            ycm-0.8.0
-           ycm-0.8.1)  # since YARP 3.0.1
+           ycm-0.8.1   # since YARP 3.0.1
+           ycm-0.8.20180628.11
```

Comment:
Missing closing `)`.

Reply:
Sorry, I forgot to fix the commit :sweat_smile:
It should compile now...